OpenRA.Mods.Common/Traits/ChangesHealth.cs

```diff
@@ -14,30 +14,32 @@
 
 namespace OpenRA.Mods.Common.Traits
 {
-	[Desc("Attach this to actors which should be able to regenerate their health points.")]
-	class SelfHealingInfo : ConditionalTraitInfo, Requires<IHealthInfo>
+	[Desc("Attach this to actors which should regenerate or lose health points over time.")]
+	class ChangesHealthInfo : ConditionalTraitInfo, Requires<IHealthInfo>
 	{
-		[Desc("Absolute amount of health points added in each step.")]
+		[Desc("Absolute amount of health points added in each step. Use negative values to apply damage.")]
 		public readonly int Step = 5;
 
 		[Desc("Relative percentages of health added in each step.",
```

Comment:
This supports negative values as well, so "Use negative values to apply damage." needs to be added here, too.

Reply:
Added.